spark/src/main/scala/org/apache/spark/sql/delta/sources/DeltaSink.scala

```diff
@@ -207,7 +205,7 @@ case class DeltaSink(
         allowStructEvolution = canMergeSchema,
         columnName = columnName
       )
-      Column(Alias(castExpr, columnName)())
+      new Column(Alias(castExpr, columnName)())
```

Comment:
Can we at least get this change in? This fixes the compliation error against the latest master branch.

Reply:
Hello @vkorukanti, @HyukjinKwon;

What is the update on this PR? I am having issue submitting a PR because of issue.